src/discof/repair/fd_repair_tile.c

```diff
@@ -424,6 +397,43 @@ fd_repair_send_requests( fd_repair_tile_ctx_t *   ctx,
 }
 
 
+static inline void
+handle_new_cluster_contact_info( fd_repair_tile_ctx_t * ctx,
+                                 uchar const *          buf,
+                                 ulong                  buf_sz ) {
+  fd_shred_dest_wire_t const * in_dests = (fd_shred_dest_wire_t const *)fd_type_pun_const( buf );
+
+  ulong dest_cnt = buf_sz;
+  if( FD_UNLIKELY( dest_cnt >= MAX_REPAIR_PEERS ) ) {
+    FD_LOG_WARNING(( "Cluster nodes had %lu destinations, which was more than the max of %lu", dest_cnt, MAX_REPAIR_PEERS ));
+    return;
+  }
+
+  /* Stop adding peers after we reach the peer max, but we may want to
+     consider an eviction policy. */
+  for( ulong i=0UL; i<dest_cnt; i++ ) {
+   if( FD_UNLIKELY( ctx->repair->peer_cnt >= FD_ACTIVE_KEY_MAX ) ) break;// FIXME: aiming to move all peer tracking out of lib into tile, leaving like this for now
+    fd_repair_peer_addr_t repair_peer = {
+      .addr = in_dests[i].ip4_addr,
+      .port = fd_ushort_bswap( in_dests[i].udp_port ),
+    };
+    int dup = fd_repair_add_active_peer( ctx->repair, &repair_peer, in_dests[i].pubkey );
+    if( !dup ) {
+      /*
+      The repair process uses a Ping-Pong protocol that incurs one
+      round-trip time (RTT) for the initial repair request. To optimize
+      this, we proactively send a placeholder Repair request as soon as
+      we receive a peer's contact information for the first time,
+      effectively prepaying the RTT cost.
+      */
```

Comment:
sorry, but it still doesn't look quite right. keep it on the same line as the `/*` and align to at most 72-columns

```suggestion
/* The repair process uses a Ping-Pong protocol that incurs one
   round-trip time (RTT) for the initial repair request. To optimize
   this, we proactively send a placeholder Repair request as soon as we
   receive a peer's contact information for the first time, effectively
   prepaying the RTT cost. */
```